docs/tasks/tls/certificate-rotation.md

```diff
@@ -0,0 +1,82 @@
+---
+approvers:
+- jcbsmpsn
+- mikedanese
+title: Certificate rotation
+---
+
+{% capture overview %}
+This page shows how to enable and configure certificate rotation for the kubelet.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
+
+* Kubernetes version 1.8.0 or later is required
+
+* Encryption at rest is beta in 1.8.0 which means it may change without notice.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Overview
+
+The kubelet uses certificates for authenticating to the Kubernetes API.
+Normally, these certificates are issued with a long expiry date, such that
+normally they do not need to be renewed.
```

Comment:
"normally" used twice; simplify: 

Normally, these certificates are issued with a long expiry date so that they do not need to be renewed.

Reply:
Done.